ansible_collections/arista/avd/roles/eos_designs/templates/base/base.py

```diff
@@ -0,0 +1,403 @@
+                return hardware
+
+    @cached_property
+    def daemon_terminattr(self):
+        cvp_instance_ip = get(self._hostvars, 'cvp_instance_ip')
+        cvp_instance_ips = get(self._hostvars, 'cvp_instance_ips')
+        if cvp_instance_ip is not None or cvp_instance_ips is not None:
+            cvp_on_prem = {'ips': []}
+            cvaas = {'ips': []}
+            if cvp_instance_ip is not None:
+                if "arista.io" in cvp_instance_ip:
+                    cvaas['ips'].append(cvp_instance_ip)
+                else:
+                    cvp_on_prem['ips'].append(cvp_instance_ip)
+            elif cvp_instance_ips is not None:
+                for cvp_ip in cvp_instance_ips:
+                    if "arista.io" in cvp_ip:
+                        cvaas['ips'].append(cvp_ip)
+                    else:
+                        cvp_on_prem['ips'].append(cvp_ip)
+
+            daemon_terminattr = {'cvaddrs': []}
+            if cvp_on_prem['ips']:
+                for cvp_instance_ip in cvp_on_prem['ips']:
+                    daemon_terminattr['cvaddrs'].append({cvp_instance_ip: get(self._hostvars,
+                                                                              'terminattr_ingestgrpcurl_port')})
+                daemon_terminattr.update({'cvauth': {'method': 'key', 'key': get(self._hostvars,
+                                                                                 'cvp_ingestauth_key')}})
+            elif cvaas['ips']:
+                for cvp_instance_ip in cvaas['ips']:
+                    daemon_terminattr['cvaddrs'].append({cvp_instance_ip: '443'})
+                    daemon_terminattr.update({'cvauth': {'method': 'token-secure',
+                                                         'token_file': get(self._hostvars, 'cvp_token_file',
+                                                                           '/tmp/cv-onboarding-token')}})
+
+            daemon_terminattr.update({'cvvrf': self._mgmt_interface_vrf})
+            daemon_terminattr.update({'smashexcludes': get(self._hostvars, 'terminattr_smashexcludes')})
+            daemon_terminattr.update({'ingestexclude': get(self._hostvars, 'terminattr_ingestexclude')})
+            daemon_terminattr.update({'disable_aaa': get(self._hostvars, 'terminattr_disable_aaa', False)})
+            return daemon_terminattr
+
+    @cached_property
+    def vlan_internal_order(self):
+        range = get(self._internal_vlan_order, 'range')
+        vlan_internal_order = {'allocation': get(self._internal_vlan_order, 'allocation'),
+                               'range': {'beginning': get(range, 'beginning'),
+                                         'ending': get(range, 'ending')}}
+        return vlan_internal_order
+
+    @cached_property
+    def event_monitor(self):
+        event_monitor = get(self._hostvars, 'event_monitor')
+        if event_monitor is not None:
+            event_monitors = {'enabled': 'true'}
```

Comment:
This is not what is in the template:
```
{% if event_monitor is defined and event_monitor == true %}
event_monitor:
  enabled: true
{% endif %}
```

here we return true if it is defined even if it is false